server_py/flatgov/bills/migrations/0001_auto_20210310_1850.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1 on 2021-03-10 18:50
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('bills', 'committeedocument'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='statement',
+            name='belongs_to',
```

Comment:
Change 'belongs_to' to 'administration'; it is a more descriptive field name and will help future developers understand what the field means.

Reply:
I created a branch with this change that you could merge into this branch.